hostinit/setup.go

```diff
@@ -666,21 +656,22 @@ func (init *HostInit) LoadClient(ctx context.Context, target *host.Host) (*LoadC
 	}
 
 	// place the binary into the directory
-	scpSetupCmd := &subprocess.ScpCommand{
-		Source:         cliBinaryPath,
-		Dest:           fmt.Sprintf("~/%s/evergreen", targetDir),
-		Stdout:         scpOut,
-		Stderr:         scpOut,
+	curlSetupCmd := &subprocess.RemoteCommand{
+		CmdString: fmt.Sprintf("cd '%s' && curl -LO '%s'",
+			fmt.Sprintf("~/%s", targetDir),
+			fmt.Sprintf("%s/%s", init.Settings.Ui.Url, hostutil.ExecutableSubPath(&target.Distro))),
```

Comment:
can we just have one fmt.Sprintf? 

I worry that we run into shell assumptions which is weird on windows.